### PR TITLE
Query multiple tables setting

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Created a table named owners
- Create a table named species
- Modified the animal's table
- Inserted data into the owners
- Inserted data into the species table
- Modified inserted animals to include the species_id value
- Modified inserted animals to include owner information
- Wrote queries (using JOIN) to answer the given questions

qm
qm2
qmm

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -6,3 +6,71 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
      ('Gabumon', '2018-11-15', 2, true,8),
      ('Pikachu', '2021-01-07', 1, false,15.04),
      ('Devimon', '2017-05-12', 5, true,11);
+
+
+-- Query and Update animals table
+
+
```
Comment (BregornOriginal) [inline data.sql]: - [ ] Kindly update the data as it's requested on the requirements 

image

File: data.sql [hunk 2]
```diff
@@ -6,3 +6,71 @@ VALUES('Agumon', '2020-02-03', 0, true,10.23),
...
+INSERT INTO animals(name,date_of_birth, escape_attempts,neutered,weight_kg,species) 
+VALUES('Charmander', '2020-02-08', 0, false,-11,NULL),
+     ('Plantmon', '2021-11-15', 2, true,-5.7,NULL),
+     ('Squirtle', '1993-04-02', 3, false,-12.13,NULL),
+     ('Angemon', '2005-06-12', 1, true,-45,NULL),
+     ('Boarmon', '2005-06-07', 7, true,20.4,NULL),
+     ('Blossom', '1998-10-13', 3, true,17,NULL),
+     ('Ditto', '2022-05-14', 4, true,22,NULL);
+
+
```
Comment (BregornOriginal) [inline data.sql:23]: - [ ] Kindly update the data as it's requested on the requirements 

image

**Review by BregornOriginal — APPROVED** [review]
Hi @Bleu-Yves-Sopoude  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ README looks professional 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Bleu-Yves-Sopoude , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Implemented the change requested to ensure all data is added in the data.sql file.
## Optional suggestions

- [ ] [optional] @Bleu-Yves-Sopoude Kindly ensure that your query results are visible in the pull request summary.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**